### PR TITLE
Provide a message on the mount directory to warn the users

### PR DESCRIPTION
**Pull request by brahmaroutu**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
Notice to users that the repo is being deprecated.

**Which issue(s) this PR fixes**:

Fixes # 

**Special notes for your reviewer**:
Please consider this as the first step to avoid users creating PR's in the repo. The new repo https://github.com/kubernetes/mount-utils is commissioned and functional. Please use that repo for further work.
Eventually this repo will be deprecated and removed.

**Release note**: none
```

```

### CODE REVIEW

**Comment by brahmaroutu** [comment]
ping @andyzhangx @msau42

**Review by msau42 — COMMENTED** [review]
/reopen

File: mount/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# WARNING ! Please read before using mount functionality
+# THIS REPOSITORY is moved : Please use https://github.com/kubernetes/mount-utils for all your work
```
[the remark this answers is not in the document]
Comment (brahmaroutu) [reply mount/README.md:2]: Fixed

**Comment by brahmaroutu** [comment]
@msau42 can I get a lgtm?

**Pull request merged**